### PR TITLE
Use iso format for config start_date and end_date, change issues book…

### PR DESCRIPTION
**Pull request by vorpal56**
Jira Key: https://pathlighthq.atlassian.net/browse/FUJ-3130 (partial)

Use iso formatted datetimes for `start_date`, `end_date`, and bookmark for `issues` for consistency. Example state that gets spit out:
```
{"type": "STATE", "value": {"bookmarks": {"issues": {"updated_at": "2023-01-10T00:37:57"}, "issue_analytics": {"updated_at": "2023-01-10T00:32:12"}}}}
```

### CODE REVIEW

**Review by ecomohr — APPROVED** [review]
lgtm

**Review by vivqu — APPROVED** [review]
lgtm as well -- just thinking through how we can make sure future tap changes also conform to this standard

Is this something we can add to tap READMEs when we go through and clean those up? (ie. "make sure bookmarks are in ISO format")

**Comment by vorpal56** [comment]
> Is this something we can add to tap READMEs when we go through and clean those up? (ie. "make sure bookmarks are in ISO format")

Yes, we should update the READMEs when we go through them. For this tap specifically, we changed it from the fork so I'm going to add a config.json.example and update the README.

File: README.md [hunk 1]
```diff
@@ -6,16 +6,31 @@ spec](https://github.com/singer-io/getting-started/blob/master/SPEC.md).
...
+    -   Issues Analytics
+    -   Messages
+    -   Apps
+    -   Agents
+-   Outputs the schema for each resource
+-   Incrementally pulls data based on the input state
+
+**config.json**
+
+```python
```
Comment (vorpal56) [inline README.md:21]: Formatted using python so I can include comments

**Pull request merged**